jsapp/js/components/submissionDataTable.es6

```diff
@@ -124,24 +125,34 @@ class SubmissionDataTable extends React.Component {
       case QUESTION_TYPES.select_one.id:
       case SCORE_ROW_TYPE:
       case RANK_LEVEL_TYPE:
-        choice = this.findChoice(data);
-        return (
-          <bem.SubmissionDataTable__value>
-            {choice.label[this.props.translationIndex]}
-          </bem.SubmissionDataTable__value>
-        );
+        choice = this.findChoice(listName, data);
+        if (!choice) {
+          console.error(`Choice not found for "${listName}" and "${data}".`);
+          return null;
```

Comment:
@magicznyleszek, quick question: if a label for a choice is missing, do we still show the raw label value ("XML name") for the response?

Reply:
Oh, we don't. Good catch!

Reply:
I've added a fallback to `|| choice.name`